src/libs/LibProvingPeriod.sol

```diff
@@ -52,7 +52,7 @@ library LibProvingPeriod {
         period.stake = stake;
     }
 
-    /// @dev Whether the period has been initialized
+    /// @dev Whether the period has been initialised
```

Comment:
nit: if you write it with a `s` here, the function name should also be with an `s`

Reply:
Applies to the rest of the instances changed in the PR